src/rust/src/x509/certificate.rs

```diff
@@ -731,6 +731,100 @@ pub(crate) fn parse_access_descriptions(
     Ok(ads.to_object(py))
 }
 
+fn parse_naming_authority<'p>(
+    py: pyo3::Python<'p>,
+    authority: NamingAuthority<'p>,
+) -> CryptographyResult<pyo3::Bound<'p, pyo3::PyAny>> {
+    let py_id = match &authority.id {
+        Some(data) => oid_to_py_oid(py, data)?,
+        None => py.None().bind(py).clone().into_any(),
```

Comment:
```suggestion
        None => py.None().into_bound(py),
```

Reply:
@alex how did I miss this in pyo3's documentation 🙈